modules/table/src/test/java/org/apache/ignite/internal/table/distributed/raft/snapshot/outgoing/OutgoingSnapshotMvDataStreamingTest.java

```diff
@@ -281,6 +280,43 @@ void sendsOutOfOrderRowsWithHighestPriority() {
         assertThat(response.rows().get(1).rowId(), is(rowId1.uuid()));
     }
 
+    @Test
+    void doesNotEnqueueMissingRows() {
+        when(mvPartitionStorage.scanVersions(rowIdOutOfOrder)).thenReturn(Cursor.fromIterator(emptyIterator()));
```

Comment:
There is a `CursorUtils#emptyCursor` method

Reply:
Switched to it. BTW, why wouldn't we have that method on the `Cursor` itself? It would increase its visibility for developers.

Reply:
We used to, but you will have to make `EMPTY_CURSOR` public, which I don't like, so I decided to move it here (like `Collections.emptyList()`. Maybe we can add a method to `Cursor` which delegates to `CursorUtils` but I don't know if it's a good idea